app/models/service/dialog_properties.rb

```diff
@@ -0,0 +1,22 @@
+class Service
+  class DialogProperties
+    require_nested :Retirement
+
+    def initialize(options, user)
+      @attributes = {}
```

Comment:
@gmcculloug Is the @attributes needed since we are parsing and just returning a hash back after the parse?